### PR TITLE
Creating new files 

### PR DESCRIPTION
**Pull request by Gojo1729**
Created a new file with new names

### CODE REVIEW

File: ga.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+as
```
Comment (Saniritvik) [inline ga.txt:1]: @Gojo1729 What's the purpose of "as"?

File: test.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+print(f"Hello World")
+print(f"Hello !!!")
```
Comment (Saniritvik) [inline test.py:2]: Why is there a second hello?

**Review by Saniritvik — CHANGES_REQUESTED** [review]
Fix the unrequired changes

File: ga.txt (hunk 1 above)
Comment (Gojo1729) [reply ga.txt:1]: For testing....

**Pull request merged**